code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -615,12 +615,15 @@
 	if(iscarbon(M) && M.stat != DEAD)
 		var/mob/living/carbon/C = M
 		var/holycheck = ishumanbasic(C)
-		if(!(holycheck || islizard(C)) || reac_volume < 5) // implying xenohumans are holy //as with all things,
+		if(!(holycheck || islizard(C) || ismoth(C)) || C.dna.features["moth_wings"] == "Burnt Off" || reac_volume < 5 ) // implying xenohumans are holy //as with all things,
```

Comment:
you dropped a ) by accident here

Reply:
Where at? It's checking if they are not humanbasic, lizard, or moth, and then if they have burnt off moth wings or if the reac_volume is <5

Reply:
![image](https://user-images.githubusercontent.com/10467687/77384184-7d52f500-6d63-11ea-8851-93c501f840b9.png)